packages/gestalt/src/shared/FormLabel.css

```diff
@@ -1,3 +1,7 @@
 .formLabel {
```

Comment:
this is changing the padding for md right?

before md and lg were 200.... is this correct?

Reply:
i think that's why all visual snapshots changed... is this a mistake?

only sm should be 100. no?

Reply:
Right, we're making a change where `md` and 'sm' have 100 now, and only large has 200